prow/pluginhelp/pluginhelp.go

```diff
@@ -18,6 +18,22 @@ limitations under the License.
 // These structs are used by sub-packages 'hook' and 'externalplugins'.
 package pluginhelp
 
+// Command is a serializable representation of the command information for a single command.
+type Command struct {
+	// Frequency suggests how often a command is used. Value ranges from 0 to 2, the SMALLER the
+	// number is, the more often the command is used.
+	Frequency int
```

Comment:
Since this isn't the actual frequency it might be better to call this something like `Priority`. I think that better aligns with what we want as well because there could be commands that aren't used frequently, but are still important enough to belong at the top of the plugin help page.
Also is there a reason for bounding the priorities to only 3 values instead of allowing any non-negative value?

Reply:
Discussed offline but we definitely shouldn't list these by usage / frequency, we'll need to surface new commands etc.